src/owl/config/configure.ml

```diff
@@ -42,6 +42,7 @@ let get_default_cflags c = [
 
 let get_default_libs c = [
   "-lopenblas";
+  "-llapacke";
```

Comment:
That would be more precise, but I think that adding this doesn't break anything in the cases where the lib is not available. Travis has passed now anyway :)

Reply:
The issue is that it is not clear which one the compiler will link to. If it is linked to binary lapacke, there might be significant performance degradation because OpenBLAS (compiled from source) offers much better performance.

Reply:
ah ok. right, i'll fix that then. but after lunch :)